aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/codegen/events/SignInEvent.kt

```diff
@@ -15,13 +15,19 @@
 
 package com.amplifyframework.statemachine.codegen.events
 
+import com.amplifyframework.auth.cognito.options.AWSCognitoAuthSignInOptions
 import com.amplifyframework.statemachine.StateMachineEvent
 import com.amplifyframework.statemachine.codegen.data.AuthChallenge
 import java.util.Date
 
 class SignInEvent(val eventType: EventType, override val time: Date? = null) : StateMachineEvent {
     sealed class EventType {
         data class InitiateSignInWithSRP(val username: String, val password: String) : EventType()
+        data class InitiateSignInWithCustom(
+            val username: String,
+            val password: String?,
+            val signInOptions: AWSCognitoAuthSignInOptions
```

Comment:
Codegen should not be dependent on Cognito package.

Reply:
Looks like its not used so removed it.